src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DocumentState.cs

```diff
@@ -110,13 +100,7 @@ async ValueTask<VersionStamp> GetTextVersionCoreAsync(CancellationToken cancella
         }
     }
 
-    public DocumentState WithConfigurationChange()
-        => new(this, _textAndVersionSource);
-
-    public DocumentState WithImportsChange()
-        => new(this, _textAndVersionSource);
-
-    public DocumentState WithProjectWorkspaceStateChange()
+    public DocumentState UpdateVersion()
```

Comment:
🥳 thank you! I always wondered why we had `With*Change` that didn't take any updated information